modules/osdk-java-generate-crd.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/java/osdk-java-tutorial.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-java-generate-crd_{context}"]
+= Generating CRD manifests
+
+After the API is defined with `MemcachedSpec` and `MemcachedStatus` files, you can generate CRD manifests.
+
+.Procedure
+
+* Run the following command from the previously created `memcached-operator` directory to generate the CRD:
++
+[source,terminal]
+----
+$ mvn clean install
+----
+
+.Verification
+
+* The CRD is stored in the `/target/kubernetes` directory. The contents of the `memcacheds.cache.example.com-v1.yml` file is shown below:
```

Comment:
Suggestion to make this more of a verification step (vs. like an output):
```suggestion
* The CRD is stored in the `/target/kubernetes` directory. Run the following to examine the contents of the `memcacheds.cache.example.com-v1.yml` file, as shown below:
+
[source,terminal]
----
$ cat memcacheds.cache.example.com-v1.yaml
----
+
.Example output
```

Reply:
I second the `.Example` label, and would go a step further to include the file and filepath. So `.Example target/kubernetes/memcacheds.cache.example.com-v1.yml` file

It is a mouthful, but I think it helps with scanability in these long documents where you are editing a lot of files.

Reply:
You could always truncate the filepath if it renders weird.